### PR TITLE
IGNITE-22747 Python Client: add tox.ini for CI/CD

### PR DESCRIPTION
**Pull request by isapego**
https://issues.apache.org/jira/browse/IGNITE-22747

### CODE REVIEW

File: modules/platforms/python/pyignite3/__init__.py [hunk 1]
```diff
@@ -45,33 +45,34 @@
...
+        return UUID
+    elif native == native_type_code.BITMASK:
+        return INT
+    elif native == native_type_code.STRING:
+        return STRING
+    elif native == native_type_code.BYTE_ARRAY:
+        return BINARY
+    elif native == native_type_code.PERIOD or native == native_type_code.DURATION:
+        return DATETIME
+    raise InterfaceError(f'Unsupported data type: {native}')
```
Comment (ptupitsyn) [inline modules/platforms/python/pyignite3/__init__.py:75]: match/case looks cleaner, do we really need to change to if/elif?

File: modules/platforms/python/tox.ini [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2021 GridGain Systems, Inc. and Contributors.
```
Comment (ptupitsyn) [inline modules/platforms/python/tox.ini:2]: Incorrect header.

**Review by ptupitsyn — CHANGES_REQUESTED** [review]

File: modules/platforms/python/pyignite3/__init__.py (hunk 1 above)
Comment (isapego) [reply modules/platforms/python/pyignite3/__init__.py:75]: Yeah, but it only works starting from Python 3.10, and we'd like support at least 3.8 and 3.9.

File: modules/platforms/python/tox.ini (hunk 2 above)
Comment (isapego) [reply modules/platforms/python/tox.ini:2]: Fixed.

**Review by ptupitsyn — APPROVED** [review]

File: modules/platforms/python/pyignite3/__init__.py (hunk 1 above)
Comment (ptupitsyn) [reply modules/platforms/python/pyignite3/__init__.py:75]: Will the CI/CD pipeline fail if I make a mistake and use Python 3.10 feature in my changes?

Comment (isapego) [reply modules/platforms/python/pyignite3/__init__.py:75]: Yeah, it will. That's how I found this issue 😄. It runs on 3.9

**Pull request merged**